github-actions/bazel/configure-remote/action.yml

```diff
@@ -12,6 +12,14 @@ inputs:
     description: |
       Whether to allow remote execution to be configured for Windows. By default,
       Windows is only configured to leverage remote caching.
+  google_credential:
+    description: |
+      A Google credential to be used as authentication for RBE API usages.
+  upload_local_results:
```

Comment:
It would probably also make sense to have a bazel configuration like `:trusted-builds` so that we can control more options like these in trusted builds?

Reply:
Updated this to be `trusted_build` instead and set a config flag.